### PR TITLE
Added 5 Icons and missing activities

### PR DESCRIPTION
**Pull request by Grabstertv**
**Icons Added:**
- Bitwarden
- Bixby
- Careem
- Tasker
- AutoInput

**Missing Activities added:**
- Calculator
- Facebook
- Messanger
- Reddit Sync
- Phone

### CODE REVIEW

File: app/src/main/res/drawable/auto__input.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <path
+        android:fillColor="#000000"
+        android:pathData="M11.176 11.2217L8.82397 9L8 9.7783L10.3521 12L8 14.2217L8.82397 15L11.176 12.7783L12 12L11.176 11.2217Z" />
+    <path
+        android:fillColor="#000000"
+        android:pathData="M15.4491 15L11 15L11 13.867L15.4491 13.867L15.4491 15Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/auto__input.xml:18]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="M10,2C9.75,2 9.5514,2.1777 9.5098,2.4277L9.1348,5.0723C8.5202,5.3223 7.9577,5.6673 7.4473,6.0527L4.959,5.0527C4.7298,4.9694 4.4688,5.0521 4.3438,5.2813L2.3535,8.7402C2.2285,8.959 2.2702,9.2292 2.4785,9.375L4.5938,11.0215C4.5417,11.334 4.5,11.6771 4.5,12C4.5,12.3229 4.5202,12.666 4.5723,12.9785L2.459,14.625C2.2715,14.7708 2.2194,15.0514 2.334,15.2598L4.334,18.7188C4.459,18.9479 4.7181,19.0202 4.9473,18.9473L7.4375,17.9473C7.9583,18.3431 8.5104,18.6777 9.125,18.9277L9.5,21.5723C9.5521,21.8223 9.75,22 10,22L14,22C14.25,22 14.459,21.8223 14.4902,21.5723L14.8652,18.9277C15.4798,18.6777 16.0423,18.3431 16.5527,17.9473L19.041,18.9473C19.2702,19.0306 19.5313,18.9479 19.6563,18.7188L21.6563,15.2598C21.7813,15.0306 21.7292,14.7708 21.5313,14.625L19.4375,12.9785C19.4792,12.666 19.5,12.3438 19.5,12C19.5,11.6667 19.4798,11.334 19.4277,11.0215L21.541,9.375C21.7285,9.2292 21.7806,8.9486 21.666,8.7402L19.666,5.2813C19.541,5.0521 19.2819,4.9798 19.0527,5.0527L16.5625,6.0527C16.0417,5.6569 15.4896,5.3223 14.875,5.0723L14.5,2.4277C14.4583,2.1777 14.25,2 14,2L10,2zM12,6.5A5.5,5.5 0,0 1,17.5 12A5.5,5.5 0,0 1,12 17.5A5.5,5.5 0,0 1,6.5 12A5.5,5.5 0,0 1,12 6.5z"
      android:strokeWidth="1.04167"
      android:fillColor="#000000"/>
  <path
      android:pathData="M9,9.5 L11.5,12 9,14.5"
      android:strokeLineJoin="round"
      android:strokeWidth="1.5"
      android:fillColor="#00000000"
      android:strokeColor="#000000"
      android:strokeLineCap="round"/>
  <path
      android:pathData="m13,14.5h2"
      android:strokeLineJoin="miter"
      android:strokeWidth="1.5"
      android:fillColor="#00000000"
      android:strokeColor="#000000"
      android:strokeLineCap="round"/>
</vector>
```

File: app/src/main/res/drawable/bitwarden.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <path
+        android:fillColor="#000000"
+        android:fillType="evenOdd"
+        android:pathData="M5 2C4.44772 2 4 2.44772 4 3V11.8548C4 14.4541 5.26292 16.8915 7.38653 18.3905L11.8857 21.5664C12.2496 21.8232 12.7394 21.8085 13.0871 21.5303L16.9976 18.402C18.8953 16.8838 20 14.5853 20 12.155V3C20 2.44772 19.5523 2 19 2H5ZM18 5L18 11.5837C18 13.7397 17.0065 15.7754 15.3069 17.1019L12.875 19V4H17C17.5523 4 18 4.44771 18 5Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/bitwarden.xml:12]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="m20.0849,3.2471c-0.1622,-0.1622 -0.3629,-0.2471 -0.5869,-0.2471L4.5019,3c-0.2239,0 -0.4247,0.0849 -0.5869,0.2471 -0.1622,0.1622 -0.2471,0.3629 -0.2471,0.5869v10c0,0.749 0.1467,1.4826 0.4324,2.2162 0.2934,0.7336 0.6486,1.3822 1.0811,1.9537 0.4324,0.5714 0.9421,1.1197 1.5367,1.6602 0.5946,0.5405 1.1429,0.9884 1.6448,1.3436 0.5019,0.3552 1.027,0.695 1.5753,1.0116 0.5483,0.3166 0.9344,0.5328 1.166,0.6409 0.2317,0.1158 0.417,0.2008 0.556,0.2625 0.1004,0.054 0.2162,0.0772 0.3398,0.0772 0.1235,0 0.2317,-0.0232 0.3398,-0.0772 0.139,-0.0618 0.3243,-0.1467 0.556,-0.2625 0.2317,-0.1158 0.6178,-0.3243 1.166,-0.6409 0.5483,-0.3166 1.0734,-0.6564 1.5753,-1.0116 0.5019,-0.3552 1.0502,-0.8031 1.6448,-1.3436 0.5946,-0.5405 1.1042,-1.0888 1.5367,-1.6602 0.4324,-0.5714 0.7876,-1.2201 1.0811,-1.9537 0.2934,-0.7336 0.4324,-1.4749 0.4324,-2.2162v-10c0,-0.2239 -0.0849,-0.417 -0.2471,-0.5869zM18.1544,13.9266c0,3.6216 -6.1467,6.7336 -6.1467,6.7336L12.0077,5.139h6.1467z"
      android:strokeWidth="0.0772201"
      android:fillColor="#000000"/>
</vector>
```

File: app/src/main/res/drawable/tasker.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/tasker.xml:15]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="M10,2C9.75,2 9.5514,2.1777 9.5098,2.4277L9.1348,5.0723C8.5202,5.3223 7.9577,5.6673 7.4473,6.0527L4.959,5.0527C4.7298,4.9694 4.4688,5.0521 4.3438,5.2813L2.3535,8.7402C2.2285,8.959 2.2702,9.2292 2.4785,9.375L4.5938,11.0215C4.5417,11.334 4.5,11.6771 4.5,12C4.5,12.3229 4.5202,12.666 4.5723,12.9785L2.459,14.625C2.2715,14.7708 2.2194,15.0514 2.334,15.2598L4.334,18.7188C4.459,18.9479 4.7181,19.0202 4.9473,18.9473L5.9004,18.5645L7.7207,15.4121A5.5,5.5 0,0 1,6.5 12A5.5,5.5 0,0 1,12 6.5A5.5,5.5 0,0 1,12.1914 6.5195L14.8242,4.7129L14.5,2.4277C14.4583,2.1777 14.25,2 14,2L10,2zM19.2285,5.0254C19.17,5.0253 19.11,5.0345 19.0527,5.0527L18.0996,5.4355L16.2793,8.5879A5.5,5.5 0,0 1,17.5 12A5.5,5.5 0,0 1,12 17.5A5.5,5.5 0,0 1,11.8086 17.4805L9.1758,19.2871L9.5,21.5723C9.5521,21.8223 9.75,22 10,22L14,22C14.25,22 14.459,21.8223 14.4902,21.5723L14.8652,18.9277C15.4798,18.6777 16.0423,18.3431 16.5527,17.9473L19.041,18.9473C19.2702,19.0306 19.5313,18.9479 19.6563,18.7188L21.6563,15.2598C21.7813,15.0306 21.7292,14.7708 21.5313,14.625L19.4375,12.9785C19.4792,12.666 19.5,12.3438 19.5,12C19.5,11.6667 19.4798,11.334 19.4277,11.0215L21.541,9.375C21.7285,9.2292 21.7806,8.9486 21.666,8.7402L19.666,5.2813C19.5723,5.1094 19.4039,5.0257 19.2285,5.0254zM14.7969,7.1582L14.6895,7.2324A5.5,5.5 0,0 1,14.7383 7.2578L14.7969,7.1582zM9.2617,16.7422L9.2031,16.8418L9.3105,16.7676A5.5,5.5 0,0 1,9.2617 16.7422z"
      android:strokeWidth="1.04167"
      android:fillColor="#000000"/>
  <path
      android:pathData="M17.866,3.8397 L7.3526,11.0495 10.3842,12.7986 6.134,20.1603 16.6474,12.9505 13.6155,11.2013Z"
      android:strokeLineJoin="miter"
      android:strokeWidth="2"
      android:fillColor="#000000"
      android:strokeColor="#00000000"
      android:strokeLineCap="butt"/>
</vector>
```

File: svgs/auto__input.svg [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+<path d="M11.176 11.2217L8.82397 9L8 9.7783L10.3521 12L8 14.2217L8.82397 15L11.176 12.7783L12 12L11.176 11.2217Z" fill="black"/>
+<path d="M15.4491 15L11 15L11 13.867L15.4491 13.867L15.4491 15Z" fill="black"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/auto__input.svg:5]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   fill="none"
   version="1.1"
   id="svg8"
   sodipodi:docname="auto__input.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs12" />
  <sodipodi:namedview
     id="namedview10"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="true"
     inkscape:zoom="16"
     inkscape:cx="0.375"
     inkscape:cy="7.53125"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg8">
    <inkscape:grid
       type="xygrid"
       id="grid1027"
       empspacing="1" />
  </sodipodi:namedview>
  <path
     id="path933"
     style="fill:#000000;stroke-width:1.04167"
     d="M 10 2 C 9.75 2 9.5514323 2.1777344 9.5097656 2.4277344 L 9.1347656 5.0722656 C 8.5201823 5.3222656 7.9576823 5.6673178 7.4472656 6.0527344 L 4.9589844 5.0527344 C 4.7298177 4.9694014 4.46875 5.0520833 4.34375 5.28125 L 2.3535156 8.7402344 C 2.2285156 8.9589844 2.2701826 9.2291667 2.4785156 9.375 L 4.59375 11.021484 C 4.541667 11.333984 4.5 11.677083 4.5 12 C 4.5 12.322917 4.5201816 12.666016 4.5722656 12.978516 L 2.4589844 14.625 C 2.2714844 14.770833 2.219401 15.051432 2.3339844 15.259766 L 4.3339844 18.71875 C 4.4589844 18.947917 4.7180989 19.020182 4.9472656 18.947266 L 7.4375 17.947266 C 7.9583334 18.343099 8.5104167 18.677734 9.125 18.927734 L 9.5 21.572266 C 9.552083 21.822266 9.75 22 10 22 L 14 22 C 14.25 22 14.458984 21.822266 14.490234 21.572266 L 14.865234 18.927734 C 15.479818 18.677734 16.042318 18.343099 16.552734 17.947266 L 19.041016 18.947266 C 19.270182 19.030596 19.53125 18.947917 19.65625 18.71875 L 21.65625 15.259766 C 21.78125 15.030599 21.72917 14.770833 21.53125 14.625 L 19.4375 12.978516 C 19.479167 12.666016 19.5 12.34375 19.5 12 C 19.5 11.666667 19.479824 11.333984 19.427734 11.021484 L 21.541016 9.375 C 21.728516 9.2291667 21.780599 8.9485678 21.666016 8.7402344 L 19.666016 5.28125 C 19.541016 5.0520833 19.281901 4.9798178 19.052734 5.0527344 L 16.5625 6.0527344 C 16.041667 5.6569011 15.489583 5.3222656 14.875 5.0722656 L 14.5 2.4277344 C 14.458333 2.1777344 14.25 2 14 2 L 10 2 z M 12 6.5 A 5.5 5.5 0 0 1 17.5 12 A 5.5 5.5 0 0 1 12 17.5 A 5.5 5.5 0 0 1 6.5 12 A 5.5 5.5 0 0 1 12 6.5 z " />
  <path
     style="fill:none;stroke:#000000;stroke-width:1.5;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     d="M 9,9.5 11.5,12 9,14.5"
     id="path1522"
     sodipodi:nodetypes="ccc" />
  <path
     style="fill:none;stroke:#000000;stroke-width:1.5;stroke-linecap:round;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     d="m 13,14.5 h 2"
     id="path2270"
     sodipodi:nodetypes="cc" />
</svg>
```

File: svgs/bitwarden.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M5 2C4.44772 2 4 2.44772 4 3V11.8548C4 14.4541 5.26292 16.8915 7.38653 18.3905L11.8857 21.5664C12.2496 21.8232 12.7394 21.8085 13.0871 21.5303L16.9976 18.402C18.8953 16.8838 20 14.5853 20 12.155V3C20 2.44772 19.5523 2 19 2H5ZM18 5L18 11.5837C18 13.7397 17.0065 15.7754 15.3069 17.1019L12.875 19V4H17C17.5523 4 18 4.44771 18 5Z" fill="black"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/bitwarden.svg:3]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   fill="none"
   version="1.1"
   id="svg4"
   sodipodi:docname="bitwarden.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs8" />
  <sodipodi:namedview
     id="namedview6"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="22.892582"
     inkscape:cx="8.8675013"
     inkscape:cy="9.2824829"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg4" />
  <g
     id="g871"
     transform="translate(0,0.99999)">
    <path
       class="st0"
       d="m 20.084945,2.24711 c -0.16216,-0.16216 -0.36294,-0.2471 -0.58687,-0.2471 H 4.501935 c -0.22394,0 -0.42471,0.0849 -0.58688,0.2471 -0.16216,0.16217 -0.2471,0.36294 -0.2471,0.58688 v 10 c 0,0.74903 0.14672,1.48262 0.43243,2.21621 0.29344,0.73359 0.64865,1.38224 1.08108,1.95367 0.43244,0.57143 0.94209,1.11969 1.53668,1.66023 0.5946,0.54054 1.14286,0.98842 1.64479,1.34363 0.50193,0.35521 1.02703,0.69498 1.57529,1.01159 0.54826,0.3166 0.93436,0.53281 1.16602,0.64092 0.23166,0.11583 0.41699,0.20077 0.55599,0.26255 0.10038,0.054 0.21621,0.0772 0.33977,0.0772 0.12355,0 0.23166,-0.0232 0.33976,-0.0772 0.139,-0.0618 0.32433,-0.14672 0.55599,-0.26255 0.23166,-0.11583 0.61776,-0.32432 1.16602,-0.64092 0.54826,-0.31661 1.07336,-0.65638 1.57529,-1.01159 0.50193,-0.35521 1.0502,-0.80309 1.64479,-1.34363 0.59459,-0.54054 1.10425,-1.0888 1.53668,-1.66023 0.43243,-0.57143 0.78764,-1.22008 1.08108,-1.95367 0.29344,-0.73359 0.43243,-1.4749 0.43243,-2.21621 v -10 c 0,-0.22394 -0.0849,-0.41699 -0.2471,-0.58688 z m -1.9305,10.67954 c 0,3.62162 -6.14672,6.73359 -6.14672,6.73359 V 4.13901 h 6.14672 z"
       id="path1124"
       sodipodi:nodetypes="cssssscssssccscsssssssscscccs"
       style="fill:#000000;stroke-width:0.0772201" />
  </g>
</svg>
```

File: svgs/tasker.svg [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+</svg>
```
Comment (patrykmichalik) [inline svgs/tasker.svg:4]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   fill="none"
   version="1.1"
   id="svg6"
   sodipodi:docname="tasker.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs10" />
  <sodipodi:namedview
     id="namedview8"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="true"
     inkscape:zoom="22.892582"
     inkscape:cx="2.7956654"
     inkscape:cy="10.549269"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg6">
    <inkscape:grid
       type="xygrid"
       id="grid918"
       empspacing="1" />
  </sodipodi:namedview>
  <path
     id="path933"
     style="fill:#000000;stroke-width:1.04167"
     d="M 10 2 C 9.75 2 9.5514316 2.1777345 9.5097656 2.4277344 L 9.1347656 5.0722656 C 8.5201816 5.3222656 7.9576816 5.6673184 7.4472656 6.0527344 L 4.9589844 5.0527344 C 4.7298184 4.9694044 4.46875 5.052083 4.34375 5.28125 L 2.3535156 8.7402344 C 2.2285156 8.958984 2.2701856 9.229166 2.4785156 9.375 L 4.59375 11.021484 C 4.54167 11.333984 4.5 11.677083 4.5 12 C 4.5 12.322917 4.5201756 12.666016 4.5722656 12.978516 L 2.4589844 14.625 C 2.2714844 14.770833 2.2194014 15.051432 2.3339844 15.259766 L 4.3339844 18.71875 C 4.4589844 18.947917 4.7180986 19.020182 4.9472656 18.947266 L 5.9003906 18.564453 L 7.7207031 15.412109 A 5.5 5.5 0 0 1 6.5 12 A 5.5 5.5 0 0 1 12 6.5 A 5.5 5.5 0 0 1 12.191406 6.5195312 L 14.824219 4.7128906 L 14.5 2.4277344 C 14.45833 2.1777345 14.25 2 14 2 L 10 2 z M 19.228516 5.0253906 C 19.170044 5.0252889 19.110026 5.0345054 19.052734 5.0527344 L 18.099609 5.4355469 L 16.279297 8.5878906 A 5.5 5.5 0 0 1 17.5 12 A 5.5 5.5 0 0 1 12 17.5 A 5.5 5.5 0 0 1 11.808594 17.480469 L 9.1757812 19.287109 L 9.5 21.572266 C 9.55208 21.822266 9.75 22 10 22 L 14 22 C 14.25 22 14.458984 21.822266 14.490234 21.572266 L 14.865234 18.927734 C 15.479818 18.677734 16.042318 18.343099 16.552734 17.947266 L 19.041016 18.947266 C 19.270182 19.030596 19.53125 18.947917 19.65625 18.71875 L 21.65625 15.259766 C 21.78125 15.030599 21.72917 14.770833 21.53125 14.625 L 19.4375 12.978516 C 19.47917 12.666016 19.5 12.34375 19.5 12 C 19.5 11.666667 19.479824 11.333984 19.427734 11.021484 L 21.541016 9.375 C 21.728516 9.2291659 21.780599 8.948567 21.666016 8.7402344 L 19.666016 5.28125 C 19.572266 5.1093747 19.403931 5.0256959 19.228516 5.0253906 z M 14.796875 7.1582031 L 14.689453 7.2324219 A 5.5 5.5 0 0 1 14.738281 7.2578125 L 14.796875 7.1582031 z M 9.2617188 16.742188 L 9.203125 16.841797 L 9.3105469 16.767578 A 5.5 5.5 0 0 1 9.2617188 16.742188 z " />
  <path
     id="path1175"
     style="fill:#000000;fill-opacity:1;stroke:none;stroke-width:2;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     d="M 17.866025,3.8397458 7.352616,11.049506 10.384245,12.79857 6.1339744,20.160255 16.64739,12.950486 13.615499,11.201293 Z"
     sodipodi:nodetypes="ccccccc" />
</svg>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Sorry for keeping you waiting! Here are some tweaks I’d suggest.

**Comment by Chefski** [comment]
Outdated icons. Stale since Nov 15 2021. Can be closed @Goooler

**Pull request closed without merging**